BUILDING_ON_WINDOWS.md

```diff
@@ -82,14 +94,29 @@ Note: This installation will take about 200 MB of disk space.
 
 Install [conan 2](https://conan.io/downloads.html) and make sure it's in your `PATH` (default setting).
 
+<details>
+   <summary>Adding conan to your PATH if you installed it with pip</summary>
+
+_Note: This will add all Python-scripts to your `PATH`, conan being one of them._
+
+1. Type "path" in the start menu and click on the "Edit the system environment variables" result that shows up (this should open up the 'System Properties' window with the 'Advanced' tab selected).
+2. Press on the "Environment Variables..." button near the bottom right of the 'System Properties' window to open up the 'Environment Variables' window.
+3. Within 'Environment Variables', look at the bottom half of the window where it lists 'System variables' and double click on the variable named 'Path' to open up the 'Edit environment variable' window.
+4. Press the 'New' button near the top right to create a new environment variable.
+5. Open up your terminal with the Visual Studio environment variables (e.g. `x64 Native Tools Command Prompt for VS 2022`) and type "where conan" to find the file path (the folder that contains the conan.exe) to add.
```

Comment:
* VS environment variables are not needed.
* `where.exe` should be used to support both `cmd` and PowerShell in one command.

```suggestion
5. Open up a terminal `where.exe conan` to find the file path (the folder that contains the conan.exe) to add.
```